src/main/java/app/coronawarn/verification/service/TanService.java

```diff
@@ -249,6 +251,11 @@ protected VerificationTan generateVerificationTan(String tan, TanType tanType, T
     return tanRepository.findByTanHash(hashingService.hash(tan));
   }
 
+  @Value("${tan.tele.valid.length}")
```

Comment:
add default  @Value("${tan.tele.valid.length:9}")

Reply:
Dont use @Value at all here - just inject VerificationApplicationConfig and use the getters :D Default Value is defined inside VerificationApplicationConfig.